### PR TITLE
don't use artifact in Java package

### LINKED ISSUE #790: artifact in package name or not?
In the maven wizard the default Java package name is 

groupId.artifactId

but if you generate at the command line it's just

groupId

by default. Which makes more sense? We should be consistent.

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh fix #790

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java [hunk 1]
```diff
@@ -383,15 +367,12 @@ private void adjustPackageName(String oldPackageName, String newPackageName) {
    * Helper function returning a suggested package name based on groupId and artifactId.
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: Let's remove `artifactId` from the comment.

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java [hunk 2]
```diff
@@ -402,12 +383,7 @@ protected static String suggestPackageName(String groupId, String artifactId) {
     // 3) Replace consecutive dots with a single dot.
     groupId = CharMatcher.is('.').trimFrom(groupId)
         .replaceAll("[^\\w.]", "").replaceAll("\\.+",  ".");
-    artifactId = CharMatcher.is('.').trimFrom(artifactId)
-        .replaceAll("[^\\w.]", "").replaceAll("\\.+",  ".");
 
-    if (!artifactId.isEmpty()) {  // No whitespace at all, so isEmpty() works.
-      return groupId + "." + artifactId;
-    }
     return groupId;
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: If you prefer, you could simplify the above lines to

```java
return CharMatcher.is('.').trimFrom(groupId)
    .replaceAll("[^\\w.]", "").replaceAll("\\.+",  ".");
```

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java [hunk 3]
```diff
@@ -383,15 +367,12 @@ private void adjustPackageName(String oldPackageName, String newPackageName) {
...
   @VisibleForTesting
-  protected static String suggestPackageName(String groupId, String artifactId) {
+  static String suggestPackageName(String groupId) {
     String naivePackageName = groupId;
-    if (!artifactId.trim().isEmpty()) {
-      naivePackageName = groupId + "." + artifactId;
-    }
 
     if (JavaPackageValidator.validate(naivePackageName).isOK()) {
       return naivePackageName;
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: If you prefer, you could simplify above by removing `nativePackageName`:

```java
  static String suggestPackageName(String groupId) {
    if (JavaPackageValidator.validate(groupId).isOK()) {
      return groupId;
    }
```

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java (hunk 2 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java (hunk 3 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: done

File: plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java (hunk 1 above)
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenAppEngineStandardWizardPage.java]: done

**Pull request merged**